### PR TITLE
Add readme file

### PR DESCRIPTION
**Pull request by OleksandrSaven**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+- **Maven**: Maven is used for project management and dependency resolution. It simplifies the build and package process.
+
+
+This project leverages these technologies and tools to provide a secure, efficient, and user-friendly online book store experience. Whether you're a developer or user, these technologies work together to make the online book store a seamless and enjoyable platform.
+## How to use
+To start this project, you'll need to follow these general steps. Keep in mind that the specific instructions may vary based on the project's structure and your environment. Here's a high-level overview of how you can start this Spring Boot-based online bookstore project:
+
+- **Prerequisites**:
+
+Ensure you have Java Development Kit (JDK) installed on your system.
```
Comment (vovasalyha) [inline README.md]: So, If I have JDK 8, I can build your project?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+Make sure you have Docker installed if you plan to run the application in a container.
+- **Clone the Repository**:
+
+Clone the project's repository from a version control system like Git.
+- **Database Configuration**:
+
+Check the project's configuration files (usually in the src/main/resources directory) to configure your database connection settings. Ensure you have a MySQL database set up and that you provide the correct credentials.
```
Comment (vovasalyha) [inline README.md]: Provide an option to run your app without manually installing MySQL and configuring a database. Provide a guide how to do all of that using Docker and docker-compose file.

**Review by vovasalyha — CHANGES_REQUESTED** [review]
GJ, but see a couple of comments. Also, it will be great if you add some interactivity to your README (some media, badges, hyperlinks etc.)

File: README.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    - Open the imported collection in Postman.
+    - Select the desired request.
+    - Click the "Send" button to execute the request.
+    - Review the response to ensure everything is working as expected.
+
+5. **Explore Endpoints**:
+
+    - The collection includes a variety of requests to different endpoints. Feel free to explore and test them to understand the API's capabilities.
+
+6. **Note**: Some requests may require authentication. Ensure you are logged in and have the necessary permissions.
```
Comment (andrii-hoienko) [inline README.md:125]: Add something about contribution

File: README.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+- **Docker**: Docker is used for containerization, making it easy to deploy and manage the application in various environments.
+
+- **Database**: The project uses a relational database MySQL to store book and user data. The choice of database can be configured to fit your requirements.
+
+- **Maven**: Maven is used for project management and dependency resolution. It simplifies the build and package process.
+
+
+This project leverages these technologies and tools to provide a secure, efficient, and user-friendly online book store experience. Whether you're a developer or user, these technologies work together to make the online book store a seamless and enjoyable platform.
+## How to use
```
Comment (andrii-hoienko) [inline README.md:33]: Mention the specific functionalities of your controllers.

**Review by andrii-hoienko — CHANGES_REQUESTED** [review]
GJ! Let`s improve your solution

File: README.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+4. Test your changes thoroughly to ensure they do not introduce new issues.
+
+5. Submit a pull request to the [main repository](https://github.com/OleksandrSaven/book-store) following the template provided.
+
+6. Your pull request will be reviewed and any necessary changes will be requested.
+
+7. Once your pull request is approved, it will be merged into the project.
+
+
+   Thank you for your interest in contributing to online-book-store! We appreciate your help in making the project better for everyone.
```
Comment (andrii-hoienko) [inline README.md]: Please avoid using 'We'. 'We' means that you worked on the project with someone else

**Review by andrii-hoienko — CHANGES_REQUESTED** [review]
Minor comment left

File: README.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+   - update order
+5. Shopping Cart Controller
+   - create cart
+   - get cart
+   - update cart 
+   - delete cart
+## API  Testing with Postman
+![](media/postman.png)
+
+We have provided a Postman collection that you can use to test the API endpoints easily.
```
Comment (andrii-hoienko) [inline README.md:99]: Please avoid 'We' in all your statements)

**Review by andrii-hoienko — APPROVED** [review]
Nice

**Pull request merged**